ui/pages/confirm-add-suggested-token/index.scss

```diff
@@ -1,9 +1,77 @@
 .confirm-add-suggested-token {
+  padding: 16px;
+
   &__link {
     @include H7;
 
     display: inline;
     color: var(--primary-blue);
     padding-left: 0;
   }
+
+  &__header {
+    @include H7;
+
+    display: flex;
+  }
+
+  &__token {
+    flex: 1;
+    min-width: 0;
+  }
+
+  &__balance {
+    flex: 0 0 30%;
+    min-width: 0;
+  }
+
+  &__token-list {
+    display: flex;
+    flex-flow: column nowrap;
+
+    .token-balance {
+      display: flex;
+      flex-flow: row nowrap;
+      align-items: flex-start;
+
+      &__amount {
+        @include H1;
+
+        color: var(--scorpion);
+        margin-right: 8px;
```

Comment:
`margin-inline-end` would be good for RTL.

Reply:
It turns out that the `.token-balance` class is not summoned here. `TokenBalance` component exists in this component. I removed these styles. We should be removing `.token-balance` from `ui/pages/confirm-import-token/index.scss` in a separate PR.

Reply:
Update: spoke with @darkwing  about this and we don't think `margin-inline-end` is necessary here. We are creating a follow-up task to update the spacing around this element after some margin/padding concerns we noticed.